examples/src/lib.rs

```diff
@@ -1,8 +1,49 @@
 use std::error::Error;
-use std::io::{Read, Write};
+use std::fs::File;
+use std::io::{BufReader, Read, Write};
 use std::net::TcpStream;
 
-use rustls::unbuffered::InsufficientSizeError;
+use rustls::client::ClientConnectionData;
```

Comment:
I would suggest we don't do this commit. Ideally, the examples here should be standalone, even if it means duplication between different examples. The idea should be that downstream users can take the entire file into their own repo, add a dependency on the `rustls` crate (and co.), and end up with something that builds.

Reply:
fixed

Reply:
I think we still have this in [50911be](https://github.com/rustls/rustls/pull/1583/commits/50911be8a80e2b6504b295516d86a9c3112e361f)

Reply:
fixed again :sweat_smile: